src/pages/cart.html

```diff
@@ -0,0 +1,197 @@
+                  </div>
+                </div>
+                <div class="ml-6 cursor-pointer px-3 py-3 hover:bg-gray-100 transition-all rounded ">
+                  <i class="fa-solid fa-trash"></i>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="cart-item cursor-pointer px-2.5 flex flex-col transition-all hover:border-gray-400 duration-200 rounded border border-transparent ">
+            <div class="flex items-center justify-between">
+              <div class="flex items-center">
+                <img src="../../assets/images/product_ring.png" alt="" class="object-cover w-20" />
+                <div class="flex flex-col justify-center">
+                  <span class="mb-3 font-openSans">
+                    Lắc tay D&G
+                  </span>
+                  <span class="text-xs text-gray-400 italic">
+                    345.000đ
+                  </span>
+                </div>
+              </div>
+              <div class="flex items-center">
+                <div class="w-24">
+                  <div class="text-base italic text-red-600 font-bold">345.000đ</div>
+                  <div class="flex justify-between items-center">
+                    <button class="hover:bg-gray-100 transition-all rounded-sm px-2.5  border">-</button>
+                    <div class="">
+                      <span class="text-xs tracking-tighter">x</span>
+                      <span class="-ml-1">1</span>
+                    </div>
+                    <button class="hover:bg-gray-100 transition-all rounded-sm px-2.5 border">+</button>
+                  </div>
+                </div>
+                <div class="ml-6 cursor-pointer px-3 py-3 hover:bg-gray-100 transition-all rounded ">
+                  <button>
+                    <i class="fa-solid fa-trash"></i>
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="text-right w-full px-2.5 font-openSans">
+        <a href="../pages/product.html"
+          class="py-2 px-3 text-white bg-black text-sm uppercase rounded-sm inline-block cursor-pointer"
+          data-i18n="cart.buy-more">
+        </a>
+        <button class="py-2 px-3 text-white bg-black text-sm uppercase rounded-sm inline-block ml-3 cursor-pointer"
+          data-i18n="cart.delete">
+        </button>
+        <a href="../pages/pay.html"
+          class="py-2 px-3 text-white bg-green-500 text-sm uppercase  rounded-sm inline-block ml-3 cursor-pointer"
+          data-i18n="cart.pay">
+        </a>
+      </div>
+    </div>
+
+    <div id="footer"></div>
+  </div>
+  <script src="../../assets/js/loadlayouts.js" type="module"></script>
+  <script src="../../assets/js/i18n.js"></script>
+  <script src="../../assets/js/breadcrumb.js"></script>
```

Comment:
```suggestion
  </div>
  
  <script src="../../assets/js/loadlayouts.js" type="module"></script>
  <script src="../../assets/js/breadcrumb.js"></script>
  <script src="../../assets/js/i18n.js"></script>
```
E để i18n xuống dưới cùng nha